youtube/services.py

```diff
@@ -0,0 +1,83 @@
+import requests
+from .models import Video
+from core.utils import db_table_exists
+from django.utils import timezone
+from core.models import User
+
+
+class YoutubeService:
+
+    @staticmethod
+    def get_user_channel_id(username, api_key):
+        r = requests.get('https://www.googleapis.com/youtube/v3/channels?part=id&forUsername=' + username
+                         + '&key=' + api_key)
+        r = r.json()
+        if not r['items']:
+            return None
+        return r['items'][0]['id']
+
+    @staticmethod
+    def get_my_videos(username, api_key, max_results=15):
+        if not(db_table_exists('youtube_video')):
+            return None
+        my_videos = []
+        if not User.is_authenticated:
```

Comment:
```
@property
    def is_authenticated(self):
        """
        Always return True. This is a way to tell if the user has been
        authenticated in templates.
        """
        return True
```
to zawsze zwraca True, to po co to dodawać? ogólnie sprawdzenie trzeba w widoku sprawdzać, a nie w API klasie